openshift/template.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: fabric8-analytics-bigquery-manifests-job
+metadata:
+  name: fabric8-analytics-bigquery-manifests-job
+  annotations:
+    description: fabric8-analytics-bigquery-manifests-job
+objects:
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: fabric8-analytics-bigquery-manifests-job
+    annotations:
+      description: fabric8-analytics-bigquery-manifests-job
+  spec:
+    successfulJobsHistoryLimit: 4
+    failedJobsHistoryLimit: 0
+    concurrencyPolicy: "Forbid"
+    schedule: "${CRON_SCHEDULE}"
+    jobTemplate:
+      spec:
+        template:
+          spec: 
+            restartPolicy: Never
+            containers:
+            - name: bigquery-manifests
+              image: "djsarkar/bigquery-manifests:latest"
+              env:
+                - name: DEBUG
+                  value: "true"
+                - name: AWS_S3_ACCESS_KEY_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: s3-access-key-id
+                      name: aws
+                - name: AWS_S3_SECRET_ACCESS_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: s3-secret-access-key
+                      name: aws
+                - name: GOOGLE_APPLICATION_CREDENTIALS
+                  valueFrom:
+                    secretKeyRef:
+                      key: bq-key
+                      name: bigquery-access
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+parameters:
+  - description: Schedule
+    displayName: Schedule
+    required: true
+    name: CRON_SCHEDULE
+    value: "*/10 * * * *"
```

Comment:
@dipanjanS this join is set to run this joib every 10 minutes as per this CRON_SCHEDULE. Please change this to the appropriate schedule.

Reply:
@samuzzal-choudhury what should be the ideal schedule do you think, can you send me the expression, i'll update this then

Reply:
@dipanjanS it can be run monthly once. The expression is going to be '0 0 1 */1 *'. This means that the job runs at 00:00 hrs on 1st day of every month.

Reply:
Thanks @samuzzal-choudhury just pushed it as a new commit now